src/action.ts

```diff
@@ -113,6 +134,8 @@ async function action(pr: PullRequest): Promise<void> {
         settings: envSettings,
         secrets: tmtEnvSecrets,
         artifacts: tmtArtifacts,
+        hardware: tmtHardware,
+        disk: tmtDisk,
```

Comment:
disk is part of the HW?

Reply:
> disk is part of the HW?

AFAIK not. based on the link https://gitlab.com/testing-farm/docs/root/-/merge_requests/120/diffs?view=inline

Reply:
Removed by commit https://github.com/sclorg/testing-farm-as-github-action/pull/146/commits/bddea9b2ace8b449d665c9889534bdb6b2497738